app/helpers/save_post_helper.rb

```diff
@@ -12,4 +12,13 @@ def joined post
     Subscription.where(community_id: post.community_id, account_id: post.account_id).exists?
   end
 
+  def selected_community communities
+    print "#############\n\n\n\n\n#################\n\n\n\n###################\n\n\n ",params[:community]
```

Comment:
please remove the print or puts before committing code

Reply:
This one is resolved